infrastructure/modules/traffic_manager/main.tf

```diff
@@ -0,0 +1,44 @@
+locals {
```

Comment:
Can locals be moved to a `locals.tf` file to keep consistent with the AKS module? (as above)